pkg/collect/cluster_resources.go

```diff
@@ -795,10 +809,15 @@ func crsV1(ctx context.Context, client dynamic.Interface, config *rest.Config, n
 
 		var version string
 		if len(crd.Spec.Versions) > 0 {
-			version = crd.Spec.Versions[0].Name
-		}
-		if len(crd.Status.StoredVersions) > 0 {
-			version = crd.Status.StoredVersions[0]
+			versions := []string{}
+			for _, v := range crd.Spec.Versions {
+				versions = append(versions, v.Name)
+			}
+
+			version = versions[0]
```

Comment:
This can just be `version = selectCRDVersionByPriority(versions)`.   There is no need for the check below.  But ok.